protocol/typescript/src/solarxr-protocol/datatypes/hardware-info/imu-type.ts

```diff
@@ -10,5 +10,8 @@ export enum ImuType {
   MPU6050 = 6,
   BNO086 = 7,
   BMI160 = 8,
-  ICM20948 = 9
+  ICM20948 = 9,
+  ICM42688 = 10,
+  BMI323 = 11,
+  LSM6DSV = 12
```

Comment:
these trackers are not yet in firmware code

Reply:
this pr can wait until those are merged then ig (also oops I didn't see the comment before)